doc/source/rst.rst

```diff
@@ -2,6 +2,25 @@
 Understanding the RST
 =====================
 
+Editing Documentation
+=====================
+
+The documentation format we use is Restructured Text, often
+abbreviated "ReST" or ".rst" as a file extension. It is very similar
+to Markdown, but more powerful since it has some extra features. But
+basically, it is just plain unformatted text. Headlines can be done
+with a "#" or "##" in front of the title, bold text is produced by
```

Comment:
this statement is wrong. Titles are managed by a line under the title text. Depending on the chosen symbol a heading level is influenced.